br/pkg/utils/cdc.go

```diff
@@ -0,0 +1,119 @@
+package utils
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.uber.org/zap"
+)
+
+const (
+	CDCPrefix      = "/tidb/cdc/"
+	ChangefeedPath = "/changefeed/info/"
+	CDCPrefixV61   = "/tidb/cdc/changefeed/info/"
+)
+
+// CDCNameSet saves CDC changefeed's information.
+// nameSet maps `cluster/namespace` to `changefeed`s
+type CDCNameSet struct {
+	nameSet map[string][]string
+}
+
+// that the nameSet is empty means no changefeed exists.
+func (s *CDCNameSet) Empty() bool {
+	return len(s.nameSet) == 0
+}
+
+// MessageToUser convert the map `nameSet` to a readable message to user.
+func (s *CDCNameSet) MessageToUser() string {
+	var changefeedMsgBuf strings.Builder
+	changefeedMsgBuf.WriteString("found CDC changefeed(s): ")
+	isFirst := true
+	for clusterID, captureIDs := range s.nameSet {
+		if !isFirst {
```

Comment:
It seems we are going to yield string like `item1, item2, item3,`, why not just append a comma after each item like:
```
for _, item := range items {
  writer.WriteString(item);
  writer.WriteString(",");
}
```

Reply:
use `isFirst` to yield string like `item1, item2, item3`.

Reply:
done